pkg/streamer/reader_test.go

```diff
@@ -589,7 +589,6 @@ func (t *testReaderSuite) TestStartSync(c *C) {
 	// NOTE: load new UUIDs dynamically not supported yet
 
 	// close the reader
-	err = r.Close()
```

Comment:
Although `Close` doesn't return error anymore, we'd better reserve it to form a complete usage flow.

Reply:
Yes, I forgot to use it, it's been fixed already in the latest commit